integration_tests/sdk/aqueduct_tests/param_test.py

```diff
@@ -518,17 +520,90 @@ def test_invalid_local_data(client):
         )
 
 
-def test_local_image_data_parameter(client, flow_name, engine):
+def test_all_local_data_types(client, flow_name, engine):
+    @op
+    def must_be_picklable(input):
+        """
+        Unable to check that the input is picklable, since `pickle.loads()`
+        complains about `import of module 'param_test' failed`.
+        """
+        print(input)
+        if input != ArtifactType:
+            raise Exception("Expected Class.")
+        return input
+
+    with open("data/test.pickle", "wb") as file:
+        pickle.dump(ArtifactType, file)
```

Comment:
To test pickle local data. I save class `ArtifactType` into a pickable file, load it as local data parameter and check for equality. 

In `test_all_param_types`, we dynamically created an `EmptyClass`, put it in `aq.create_param()`, and check for equality.

I didn't do what `test_all_param_types` did is because `pickle.load` complain a lot in pytest. Would that be enough of coverage.

Reply:
Can you share the error you were seeing after doing what `test_all_param_types` does.

Reply:
I'd rather we try to test this the same way of creating an empty class

Reply:
<img width="1146" alt="Screen Shot 2023-04-07 at 3 09 09 PM" src="https://user-images.githubusercontent.com/78303449/230685098-10dec96f-282f-4113-a232-5fe864c9ffa8.png">
I have been  researching on this error and a lot of the complaints are coming from using multiprocessing package. I suspect that pytest is also using some sort of multiprocessing which causes the pickle.dump to fail?

Reply:
I have created https://linear.app/aqueducthq/issue/ENG-2774/investigate-the-limitation-of-artifacttypepicklable-from-create-param. Feel free to give it a read to see if the issue makes sense